src/OverlayTrigger.js

```diff
@@ -281,19 +281,21 @@ const OverlayTrigger = React.createClass({
 
   _getContainerDimensions() {
     const containerNode = this.getContainerDOMNode();
-    let width, height;
+    let width, height, scroll;
+
     if (containerNode.tagName === 'BODY') {
       width = window.innerWidth;
       height = window.innerHeight;
+      scroll =
+        domUtils.ownerDocument(containerNode).documentElement.scrollTop ||
```

Comment:
eventually this line sometimes is:
```js
domUtils.ownerDocument(domUtils.ownerDocument(this).body).documentElement.scrollTop ||
```

because of:
```js
// domUtils.js
function ownerDocument(componentOrElement) {
  let elem = React.findDOMNode(componentOrElement);
  return (elem && elem.ownerDocument) || document;
}
```
```js
// OverlayMixin.js
getContainerDOMNode() {
  return React.findDOMNode(this.props.container) || domUtils.ownerDocument(this).body;
}
```
```js
// this file
const containerNode = this.getContainerDOMNode();
...
domUtils.ownerDocument(containerNode).documentElement.scrollTop ||
```

It seems there are some technical debt exists in those files already and you just trap into it.

Have you seen this ? http://stackoverflow.com/a/20478983/3874570
```js
function getScrollTop() {
  return window.pageYOffset || document.documentElement.scrollTop || document.body.scrollTop || 0;
};
```
And this ? https://miketaylr.com/posts/2014/11/document-body-scrollTop.html
```js
(document.documentElement ||
 document.body.parentNode ||
 document.body).scrollTop;
```
I hope it will do some help. :cherries: